bitnami/common/templates/_images.tpl

```diff
@@ -0,0 +1,38 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
For now it's ok, let's see what happens with the new Pivotal convention

Reply:
yes!